Asterix/utils/save_and_read.py

```diff
@@ -39,6 +39,9 @@ def save_plane_in_fits(dir_save_fits, name_plane, image):
     current_time_str = datetime.datetime.today().strftime('%H_%M_%S_%f')[:-3]
     name_fits = current_time_str + '_' + name_plane
 
+    if os.path.exists(dir_save_fits) is False:
```

Comment:
I suspect this is in case somebody doesn't read the docstring and assumes `dir_save_fits` is a boolean? That looks ok.

However the correct way to check if a bool is true is just:
```python
if my_boolean:
    pass
```
and to check whether it is false:
```python
if not my_boolean:
    pass
```
otherwise you're technically not checking the bool value but just making a comparison which can give unexpected results (search for "Don’t compare boolean values to True or False using" in link):
https://peps.python.org/pep-0008/#programming-recommendations

Reply:
I know, I just created a branch tonigth to correct those (Small pylint warning corrections). It branch out of this one so let's do that one first